### ISSUE TITLE
Make keymaps and commands local to a debug view

### ISSUE THREAD

**Issue by sakhnik** [sakhnik/nvim-gdb#14]
The debugging commands don't make sense outside the debugging session. Set/unset them as the user navigates the tab pages.
Alternatively, make mappings buffer-local. Keep in mind that a user may open buffers in the debug tab page too.

**Comment by ploth** [comment]
After quitting a debug session and returning to my tab some keymaps are still overwritten like `Ctrl - p`

**Comment by sakhnik** [comment]
@ploth, unfortunately, I can't reproduce the issue. But please feel free to open another one describing the exact steps to reproduce. It would be helpful if neovim were run like so: `cd test; nvim -u init.vim`

**Comment by ploth** [comment]
> It would be helpful if neovim were run like so: cd test; nvim -u init.vim

Do you want me to start over with an empty init file?

I don't know exactly how to debug this issue/help you with my information.
After starting nvim I usually press `ctrl - p` to search for the file I want to edit.
`:map <C-p>` results in the following output: `n  <C-P>       * :FZF<CR>`
Typing `:GdbStart gdb -q -f <mybinary>` starts the the TUI like environment in a new tab.
After my debug session I write `quit` in the debuggers console. It is not necessary to do anything. Writing `quit` instantly results in the same issue.
This results in switching to my previous tab where I edited my file.
All keymaps are overwritten from now on.
`:map <C-p>` results in the following output: `No mapping found`

These are my settings for this plugin:

```nvim
" nvim-gdb ((((((((((((((((((((((((((((((((((((((((((((((((
" gdb and lldb wrapper
Plug 'sakhnik/nvim-gdb'
let g:nvimgdb_disable_start_keymaps = 1
" nvim-gdb ))))))))))))))))))))))))))))))))))))))))))))))))
```

PS: Had the same issue before setting `let g:nvimgdb_disable_start_keymaps = 1`

**Comment by sakhnik** [comment]
Thanks, now I understand what happened, and will be able to fix the issue #19 soon.
